### PR TITLE
Dynamically test javascript URL string return values in frames

### PR DESCRIPTION
**Pull request by rwlbuis**
Test javascript URL string return values in direct and indirect (target) frame contexts.

### CODE REVIEW

File: html/browsers/browsing-the-web/navigating-across-documents/javascript-url-return-value-handling-dynamic.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    href.target = "hi" + input;
+    href.href = javascriptURL;
+    t.step_timeout(() => {
+      assert_equals(frame.contentDocument.body.textContent, output);
+      assert_equals(frame.contentDocument.charset, document.charset);
+      t.done();
+    }, 200)
+    document.body.appendChild(frame);
+    document.body.appendChild(href);
+    href.click();
```
Comment (annevk) [inline html/browsers/browsing-the-web/navigating-across-documents/javascript-url-return-value-handling-dynamic.html:45]: Wouldn't this ruin the entire harness in user agents that don't use the `target` attribute?

Comment (rwlbuis) [reply html/browsers/browsing-the-web/navigating-across-documents/javascript-url-return-value-handling-dynamic.html:46]: I was not ware Firefox did not support it. Anyway latest update should be better as it splits up the frame scenario and the link target scenario.

Comment (annevk) [reply html/browsers/browsing-the-web/navigating-across-documents/javascript-url-return-value-handling-dynamic.html:46]: Firefox does, but Chrome/Safari don't, right?

Comment (rwlbuis) [reply html/browsers/browsing-the-web/navigating-across-documents/javascript-url-return-value-handling-dynamic.html:48]: Yes, that is correct, and I have a fix ready for WebKit/Safari.
I noticed an error in my change, should be fixed now.

**Review by annevk — APPROVED** [review]

**Pull request merged**